yak/src/mc/marching_cubes.cpp

```diff
@@ -39,7 +39,7 @@ std::vector<Triangle> processCube(const yak::TSDFContainer& grid, int x, int y,
     return 5.0f * float(f);
   };
 
-  const static uint16_t min_weight = 4;
+  const static uint16_t min_weight = 1;
```

Comment:
Suggestion: make this a parameter / configurable.

`4` was an assumption which turned out not to work for all cases, but `1` could also suffer from this (it may actually be good for YAK to 'ignore' certain voxels, as a sort of filter).

Allowing users to configure this will make this possible.

Reply:
I think making this a parameter makes sense. With higher framerate cameras it actually makes sense to skip voxels with a low number of observations, as those are probably regions that are not of interest to the user. 

I will have a look at making this a parameter that we can set.

Reply:
@dave992 Since this primarily affects your application, I'll give you some time to explore alternative solutions rather than rushing to merge this in.